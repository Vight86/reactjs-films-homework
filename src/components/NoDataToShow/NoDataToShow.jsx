import React from 'react';
import styles from './NoDataToShow.scss';

const NoDataToShow = () => (
  <div className={styles.noMoviesWrapper}>
    <div className={styles.noMoviesImage} />
    <p className={styles.noMoviesTitle}>
      We&#39;re sorry. We have &#39;t found any movies to match your request.
    </p>

  </div>
);

export default NoDataToShow;
