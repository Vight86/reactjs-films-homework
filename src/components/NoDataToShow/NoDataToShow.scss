@import '../../utils/variables';

.noMoviesWrapper {
  display: flex;
  flex-direction: column;
  align-items: center;
}

.noMoviesTitle {
  color: $color-secondary;
  font-size: 4em;
  line-height: auto;
  text-align: center;
}

.noMoviesImage {
  width: 96%;
  max-width: 400px;
  padding-top: min(400px, 96%);
  margin: 10px auto;
  background-image: url('no-movies.svg');
  background-position: top center;
  background-repeat: no-repeat;
  background-size: contain;
}
