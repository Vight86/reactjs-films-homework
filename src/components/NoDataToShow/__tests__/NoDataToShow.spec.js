import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import NoDataToShow from '../index';

let renderer;
let result;

beforeEach(() => {
  renderer = new ShallowRenderer();
  renderer.render(<NoDataToShow />);
  result = renderer.getRenderOutput();
});

afterEach(() => {
  renderer = null;
  result = null;
});

describe('render NoDataToShow component', () => {
  it('render component correctly', () => {
    expect(result).toMatchSnapshot();
  });
});
