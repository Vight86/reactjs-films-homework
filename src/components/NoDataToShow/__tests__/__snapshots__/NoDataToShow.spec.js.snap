// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render NoDataToShow component render component correctly 1`] = `
<div
  className="noMoviesWrapper"
>
  <div
    className="noMoviesImage"
  />
  <p
    className="noMoviesTitle"
  >
    We're sorry. We have 't found any movies to match your request.
  </p>
</div>
`;
